primitives/ls.yaml

```diff
@@ -1175,16 +1175,16 @@ definitions:
   #INTEL - AVX512
   - target_extension: "avx512"
     ctype: [ "uint32_t", "int32_t", "uint64_t", "int64_t" ]
-    lscpu_flags: [ "avx512f" ]
+    lscpu_flags: [ "avx512f", "avx512vbmi2" ]
     implementation: "_mm512_mask_compressstoreu_epi{{ intrin_tp[ctype][1] }}(reinterpret_cast<void*>(memory), mask, data);"
   - target_extension: "avx512"
     ctype: ["float", "double"]
-    lscpu_flags: [ "avx512f" ]
+    lscpu_flags: [ "avx512f", "avx512vbmi2" ]
```

Comment:
Float and double is also supported directly by avx512f ([1])

[1] https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=_mm512_mask_compressstoreu_&ig_expand=1490,1493